pyfar/constants/constants.py

```diff
@@ -434,3 +435,116 @@ def density_of_air(
 
     # Equation 6.71
     return P / (R_a * (temperature + 273.15)) * (1+C)/(1+alpha*C)
+
+
+def octave_band_tolerance(
+        exact_center_frequency: float,
+        bands: Literal["octave", "third"],
+        tolerance_class : Literal[1, 2]):
+    r"""
+    Calculate the tolerance limits for fractional octave band filters.
+
+    Calculation is in accordance with IEC 61260-1:2014 [#]_ (Section 5.10 and
+    Table 1).
+
+    .. note ::
+        The standard defines some lower tolerance limits to :math:`-\infty`,
+        which is inconvenient for plotting. The returned tolerance is -60000 dB
+        in these cases, which is below the smallest possible value approx.
+        -6000 dB (``20*np.log10(np.finfo(float).tiny)``).
+
+    Parameters
+    ----------
+    exact_center_frequency : float
+        The exact center frequency of the band filter in Hz (see
+        :py:func:`~pyfar.dsp.filter.fractional_octave_frequencies`).
+    bands : Literal["octave", "third"]
+        The type of fractional octave band. Must be either 'octave' or 'third'.
+    tolerance_class : Literal[1, 2]
+        The tolerance class as defined in the standard. Must be 1 or 2.
+
+    Returns
+    -------
+    tolerance : FrequencyData
+        Tolerance limits in dB. The lower tolerance is given in the first
+        channel (``tolerance.freq[0]``) and the upper tolerance in the second
+        (``tolerance.freq[1]``).
```

Comment:
even if the standard is just defininig it in dB, the pyfar style would be e.g. for plotting to also return it as an energy FrequencyData object.

Reply:
We will do it as suggested but not energetic.

Reply:
as discussed in the weekly, returning db values and frequencies in separate numpy arrays